backend/src/v5/models/teamspaces.js

```diff
@@ -145,6 +145,13 @@ Teamspace.createTeamspaceSettings = async (teamspace) => {
 	await db.insertOne(teamspace, 'teamspace', settings);
 };
 
+Teamspace.getAllUsersInTeamspace = async (teamspace, projection = {}) => {
+	const query = { 'roles.db': teamspace, 'roles.role': TEAM_MEMBER };
+	const users = await findMany(query, projection);
```

Comment:
is there much point in taking in a projection if this function only returns a list of user names?

```suggestion
	const users = await findMany(query, { user: 1});
```
```suggestion
	const users = await findMany(query, projection);
```